vowpalwabbit/example.h

```diff
@@ -55,6 +57,7 @@ typedef union {
   v_array<float> scalars;           // a sequence of scalar predictions
   ACTION_SCORE::action_scores a_s;  // a sequence of classes with scores.  Also used for probabilities.
   uint32_t multiclass;
+  CCB::decision_scores_t decision_scores;
```

Comment:
It looks to me that `decision_scores_t d_s` would be more consistent with `action_scores a_s`.